__tests__/utils/operations.js

```diff
@@ -738,19 +738,23 @@ module.exports = {
           realmConfig.apiUrl
         )
         const user = realm.fromObject(userJSON)
-        return user.shareSecretWithUsername(
-          secretName,
-          secretType,
-          usernameToAdd
-        )
+        return user
+          .shareSecretWithUsername(secretName, secretType, usernameToAdd)
+          .then(function(secret) {
+            if (secret != null) {
+              return JSON.stringify(secret)
+            } else {
+              return secret
+            }
+          })
       },
       JSON.stringify(config),
       user.stringify(),
       secretName,
       secretType,
       usernameToAdd
     )
-    return result
+    return JSON.parse(result)
```

Comment:
Is this just a plain object? Should it parse to a type at all?

Reply:
its a plain object with a recordType, encrypted access key and public key and

Reply:
I've been thinking we may want to create actual secret types at some point, but we can revisit this then.